### PR TITLE
Fix CI by disabling aufs tests

### PR DESCRIPTION
**Pull request by zed3250**

### CODE REVIEW

**Comment by zed3250** [comment]
The fix is to skip aufs tests because, for some reason, the tests detect that the github agents support aufs when they don't

**Comment by Lqp1** [comment]
Can you repush the branch with only the commit that interests us please ? And fix the name of the PR, that will make things clearer

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -36,6 +36,9 @@ jobs:
           name: mesos-built-${{ github.sha }}
           path: build/mesos-built-${{ github.sha }}.tar.gz
 
+      - name: install aufs
```
Comment (Lqp1) [inline .github/workflows/ci.yml]: Do we need this then ?

**Review by Lqp1 — DISMISSED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (zed3250) [reply .github/workflows/ci.yml]: nope, leftover. thanks for spotting it!

**Review by Lqp1 — APPROVED** [review]

**Pull request merged**